<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DriverControlsTab</class>
 <widget class="QWidget" name="DriverControlsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="regenBraking">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="driverInputsGroup">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Driver Inputs</string>
   </property>
   <widget class="QCheckBox" name="brakes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Brakes</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="forward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Forward</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reverse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pushToTalk">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Push to Talk</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="horn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Horn</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="aux">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aux</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_85">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Regen Braking</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="lightsInputsGroup">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Lights Inputs</string>
   </property>
   <widget class="QCheckBox" name="headlightsOff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Headlights Off</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="headlightsLow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Headlights Low</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="headlightsHigh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Headlights High</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="signalRight">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal Right</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hazard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hazard</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="signalLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal Left</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="interior">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Interior</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="acceleration">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="musicInputGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Music Inputs</string>
   </property>
   <widget class="QCheckBox" name="volumeUp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume Up</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="volumeDown">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume Down</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="nextSong">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Song</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="previousSong">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous Song</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="alive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alive</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_84">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Acceleration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
